### PR TITLE
add navigation selection, fix filter box and table sort icon, add circle svg

### PR DESCRIPTION
**Pull request by pochretien**

### CODE REVIEW

File: docs/_components/buttons/01_button.html [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ---
 type: buttons
 title: Base button
+url: base_button
```
Comment (marilynechan) [inline docs/_components/buttons/01_button.html]: Can't you generate a URL from the title?
Something like `toLowerCase().replace(/[^\s], '_')`

File: docs/_includes/component_header.html [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<header class="sg-component-header">
+<header id="{{ entry.url}}" class="sg-component-header">
```
Comment (marilynechan) [inline docs/_includes/component_header.html]: Formatting: add a space after url.
However, as I said in an earlier comment, this should probably be generated.

**Comment by marilynechan** [comment]
Woops, build failed!
Other than that and my comment on the not generated url, nice work!

**Comment by wfortin** [comment]
LGTM

**Comment by marilynechan** [comment]
Ship it!

**Pull request merged**